bobtemplates/plone/portlet/portlets/+portlet_name_normalized+.pt.bob

```diff
@@ -1,9 +1,17 @@
 <div class="weather">
     <h3>Weather Report</h3>
-    ${view/weather_report}
+    <span tal:content="view/weather_report">
+        Error in fetching weather data
+        </span>
     <br>
-    Pressure: ${view/get_pressure}
+    Pressure: 
+    <span tal:content="view/get_pressure">
+        Cannot fetch pressure data
```

Comment:
this content is useless, you will never see it. If you want this string to be seen, when the fetching is not working, it has to done inside the view class. What ever the method is returning will be inserted into this element.